app/import.go

```diff
@@ -42,47 +42,48 @@ func (a *App) BulkImport(fileReader io.Reader, dryRun bool, workers int) (*model
 		var line LineImportData
 		if err := decoder.Decode(&line); err != nil {
 			return model.NewAppError("BulkImport", "app.import.bulk_import.json_decode.error", nil, err.Error(), http.StatusBadRequest), lineNumber
-		} else {
```

Comment:
This remove, the else, and unindent everything inside it. Then, for the clause `lineNumber == 1` it do the version import and then "continue", and any other case, get unindented too.